### DISCUSSION TITLE
스프린트 4 기능

### DISCUSSION THREAD

**Discussion by Byeongju-Kong** [woowacourse-teams/2022-sokdak#389] [Ideas]
# 스프린트 4 기능

어떤 것을 더 붙히면 좋을지 공유해주세요~!

### 공통
- 매 반복주기마다 FE에서 BE까지 관통하는 기능 중 최소 하나를 FE/BE 개발자가 짝(3인 짝도 가능) 프로그래밍으로 구현하고 데모할 때 공유
- 프로덕션(실제 서비스 되고 있는) DB 데이터를 drop할 수 없음
### 프론트엔드
- 성능 리포트를 공유한다.
- [요구사항 상세 가이드](https://techcourse.woowahan.com/s/r4gVw5VN/lt/B31r5JRJ)
- 테스트 자동화
### 백엔드
- 정적 분석 리포트를 공유한다.
- CloudWatch logs 대시보드를 구성한다.

**Comment by Byeongju-Kong** [comment]
### 마이페이지
- 닉네임 수정
- 내가 쓴 글 모아보기
- 댓글 알림 ( 푸쉬 알림 아닙니다 )
### 대댓글
- 많은 사람들이 원해보입니다.
### 포수타
포수타를 관리를 잘 해야할 것 같습니다.
포수타의 경우에는 2주에 한번? 1주에 한번? 오기 떄문에,
2주에 한번, 1주에 한번 새로운 섹션을 열어도 좋을 것 같아요.
예를 들어서 다음주 금요일에 네수타가 있다면, `2022.08.12(금) 네수타` 와 같은 섹션을 어드민이 열면 좋을 것 같아요

### 간단한 공지사항
- 메인페이지에 공지사항 버튼을 눌러서 사람들에게 좋은 커뮤니티 문화를 만들어돌라는 말을 쓰면 좋을 듯 합니다

### 아이디, 닉네임
- 사람들이 아이디를 본인이 쓰던 것을 사용해서 회원 정보 테이블을 열어보면 누군지 식별할 수 있음. 따라서, 회원 정보에 있는 모든 데이터를 해싱하는 것이 좋아보임.
- 
<img width="733" alt="스크린샷 2022-08-06 오후 11 20 43" src="https://user-images.githubusercontent.com/68771917/183252897-8f5c7eb7-df8b-49eb-8fb8-9a5eb29d806a.png">

오타 수정해야함

**Comment by byhhh2** [comment]
- 닉네임 수정
- 아이디와 비밀번호를 잃어버린 사람을 위한 폼 작성란
- 프로필 페이지
- 대댓글 
- 댓글 좋아요

**Comment by JUDONGHYEOK** [comment]
- 프로필
- 대댓글
- 알림창

**Comment by hyunrrr** [comment]
- 대댓글
- 읽은글 표시
- @josh << 알람기능
- 닉네임 수정

**Reply by injoon2019** [reply]
조시 진짜로 알림 보냈네.. 심지어 유명한 사람이네

**Comment by injoon2019** [comment]
- 프로필 기능
- 회원 가입 기능 사용성 개선
- 대댓글
- 알림

**Comment by EastHShin** [comment]
- 대댓글 기능
- 조회수 기능(한 사용자당 한번의 조회수만 허용할 것인지? 중복 조회를 허용할 것인지?)
- cloudWatch 붙이기
- 검색기능
- 맡은 도메인별 리팩터링
- 테스트 커버리지 높이기

**Comment by BETTERFUTURE4** [comment]
- 아이디/비밀번호 단방향 해싱!
- 사진 추가 기능
- 프로필
- 대댓글
- 검색 기능
- 조회수 기능
- 닉네임 수정
